summary_workflows/identification/nextflow.config

```diff
@@ -92,12 +96,12 @@ params  {
   otherdir = "${params.output}/other"
 
   // other parameters
-  windows = "100 50 10" // type int, several values inside string separated by spaces. The first value is used for relative PAS usage calculation.
-  genome_dir = "${params.data_dir}/ref/genomes" // Must contain a .gtf file with same substring as in challenge_ids, e.g. test.genome.mm10_test.gtf
+  windows = "100 50 25 10" // type int, several values inside string separated by spaces. The first value is used for relative PAS usage calculation.
+  genome_dir = "${params.data_dir}/genomes" // Must contain a .gtf file with same substring as in challenge_ids, e.g. test.genome.mm10_test.gtf
   offline = 1 // type int; 0 for False
 
   //set DEFAULT_eventMark
-  event_date = '2022-10-28'
+  event_date = '2023-03-23'
```

Comment:
I think here we have to make sure that all our runs from all events contain the same date, as otherwise it could be that they will be visualized in OEB separately (not compared)